tests/tracking/test_mlflow_artifacts.py

```diff
@@ -245,3 +245,58 @@ def test_mlflow_artifacts_example(tmpdir):
         check=True,
         cwd=os.path.join(root, "examples", "mlflow_artifacts"),
     )
+
+
+def test_rest_tracking_api_list_artifacts_with_proxied_artifacts(artifacts_server, tmpdir):
+    def list_artifacts_via_rest_api(url, run_id, path=None):
+        if path:
+            resp = requests.get(url, params={"run_id": run_id, "path": path})
+        else:
+            resp = requests.get(url, params={"run_id": run_id})
+        resp.raise_for_status()
+        return resp.json()
+
+    url = artifacts_server.url
+    mlflow.set_tracking_uri(url)
+    api = f"{url}/api/2.0/mlflow/artifacts/list"
```

Comment:
Given the MLflow Tracking URI tends to be the base for the artifact URI(url is passed through here for that reason)

Is there a way to deploy a proxied artifact server with artifact uri mlflow-artifacts, that gets red client side into
Tracking UI + /api/2.0/mlflow/artifacts

Effectively, can we make this short cut you did for testing a supprted feature in the client + mlflow server call?

Reply:
Hi Eddie! Thanks for asking about this - we considered using `api/2.0/mlflow/artifacts` as the base but opted to go with `mlflow-artifacts` to distinguish the API surface for proxied artifact uploads from the API surface for MLflow Tracking. If we included these in the MLflow Tracking surface, all implementers of MLflow Tracking would be required to support it.